src/xref/expand.ml

```diff
@@ -914,13 +914,19 @@ let expand_module_type t mty =
         let expansion = force_expansion t root (expand_module_type t root mty) in
           { mty with expansion }
 
+let remove_docs_from_signature s =
+  (* Remove bare doc comments from a signature *)
+  let open Signature in
+  List.filter (function | Comment (`Docs _) -> false | _ -> true) s
+
 let expand_include t incl =
   let open Include in
     if incl.expansion.resolved then incl
     else begin
       let root = Identifier.Signature.root incl.parent in
         match expand_include t root incl with
-        | Expanded content ->
+        | Expanded content' ->
+            let content = remove_docs_from_signature content' in
```

Comment:
That doesn't look right. Aren't you removing all the the floating doc comments from that signature? I think we just want to remove the first one.

Reply:
It's just the first that wants to go is it? I wasn't completely sure. I'll push an update.